test_libs/pyspec/eth2spec/test/block_processing/test_process_attestation.py

```diff
@@ -0,0 +1,301 @@
+from copy import deepcopy
```

Comment:
Had to manually check this file against the `dev` branch file due to file location move.

Ported tests from dev:
* `test_success_since_max_epochs_per_crosslink`

Had a to modify a few due to naming/etc